stubs/docker/docker/models/containers.pyi

```diff
@@ -71,18 +71,32 @@ class Container(Model):
         follow: bool | None = None,
         until: datetime.datetime | float | None = None,
     ) -> bytes: ...
-    def pause(self): ...
+    def pause(self) -> None: ...
     def put_archive(self, path: str, data) -> bool: ...
-    def remove(self, **kwargs) -> None: ...
+    def remove(self, *, v: bool = False, link: bool = False, force: bool = False) -> None: ...
     def rename(self, name: str): ...
     def resize(self, height: int, width: int): ...
-    def restart(self, **kwargs): ...
-    def start(self, **kwargs) -> None: ...
+    def restart(self, *, timeout: float | None = 10): ...
+    def start(self) -> None: ...
```

Comment:
See https://github.com/python/typeshed/pull/12077/files#r1623166228 for why I chose to remove `**kwargs`